.github/workflows/deploy-cloud-functions.yml

```
@@ -0,0 +1,75 @@
+          --trigger-topic token-refresh \
+          --region ${{ env.REGION }} \
+          --source ./cloud_functions/token_refresh \
+          --entry-point refresh_token \
+          --allow-unauthenticated \
+          --memory 128MB \
+          --timeout 60s \
+          --min-instances 0 \
+          --max-instances 10 \
+          --set-env-vars PROJECT_ID=${{ env.PROJECT_ID }} \
+          --vpc-connector projects/${{ env.PROJECT_ID }}/locations/${{ env.REGION }}/connectors/your-vpc-connector \
+          --service-account ${{ secrets.GCP_SA_EMAIL }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)